Clean-ups in BiT-ResNet model documentation and edits to my author page

sayakpaul opened the pull request:
Any pull request you open is subject to the TensorFlow Hub Terms of Service at www.tfhub.dev/terms and Google's Privacy Policy at https://www.google.com/policies/privacy.

Please provide only one commit. You can continuously add changes to your latest commit using [`git commit --amend`](https://stackoverflow.com/q/179123) followed by `git push -f` to update your remote branch. Alternatively, [here](https://stackoverflow.com/q/5189560) are details on how to squash multiple commits into one commit.

We check modified Markdown files for validity using [this](https://github.com/tensorflow/tfhub.dev/blob/master/.github/workflows/contributions-validator.yml) GitHub Workflow. You can execute the same checks locally by passing the file paths of modified Markdown files (relative to the `assets/docs` directory) to validator.py e.g. `python3 ./tools/validator.py google/google.md google/models/albert_base/1.md ...`.

sayakpaul commented:
@MorganR this should be fixed.

MorganR commented on assets/docs/sayakpaul/collections/bit-resnet/1.md:
```diff
@@ -36,7 +36,11 @@ Models included in this collection have two variants: (1) off-the-shelf inferenc
 ## References 
```
Please double check the references here to be sure they match the model names. It looks like the classification models should say `distill_bit_resnet50x1`, while the feature extractors are `distill_bit_r50x1`. Also note that the classification models have been linked twice while the feature extractors are missing.

MorganR commented on assets/docs/sayakpaul/bit_resnet152x2_384_feature_extraction/1.md:
```diff
@@ -1,6 +1,6 @@
 # Module sayakpaul/bit_r152x2_384_feature_extraction/1
```
And here

MorganR commented on assets/docs/sayakpaul/bit_resnet152x2_224_feature_extraction/1.md:
```diff
@@ -1,6 +1,6 @@
 # Module sayakpaul/bit_r152x2_224_feature_extraction/1
```
And here

MorganR reviewed (changes requested):
Your new changes look good, but I'm commenting with a few other changes mainly to fix issues with the links in the collection.

sayakpaul commented:
Thanks, @MorganR! I have addressed the comments. My bad regarding the mistakes introduced in the filenames of the feature extractor. Is it possible to cover this scenario from the validator script?

MorganR reviewed (approved):
Thank you for fixing the issues! Agreed on checking this in the validator. We've already filed an internal FR to address this.
